chia/pools/pool_wallet.py

```diff
@@ -966,7 +967,9 @@ async def get_pending_change_balance(self) -> uint64:
     async def get_max_send_amount(self, records: Optional[Set[WalletCoinRecord]] = None) -> uint128:
         return uint128(0)
 
-    async def coin_added(self, coin: Coin, height: uint32, peer: WSChiaConnection) -> None:
+    async def coin_added(
+        self, coin: Coin, height: uint32, peer: WSChiaConnection, coin_data: Optional[Streamable]
```

Comment:
As above.  Also the protocol check at the beginning of the class needs a parameter for the generic like the others.